docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+are currently only supported by the `CRFEntityExtractor` and the
+`DIETClassifier` component! Hence, other entity extractors, like
+`MitieEntityExtractor` or `SpacyEntityExtractor` won't use the generated
+features and their presence will not improve entity recognition for
+these extractors. Currently, all intent classifiers make use of
+available regex features.
+
+:::note
+
+Regex features only define entities when used in combination with the `RegexEntityExtractor`. Otherwise they  
+don't define entities nor intents! They only provide patterns to
+help the classifier recognize entities and related intents. Hence, you
+still need to provide intent & entity examples as part of your
+training data!
+:::
+
+### Lookup Tables
+
+Lookup tables provide a convenient way to supply a list of entity
+examples. The format is as follows:
+
+```yaml
+- lookup: <lookup table name>
+  examples: |
+  - <an entity>
+  - <another entity>
+```
+
+The name of the lookup table is subject to the same constraints as the
+name of a regex feature.
+
+When lookup tables are supplied in training data, the contents are
+combined into a large regex pattern that looks for exact matches in the
+training examples. These regexes match over multiple tokens, so
+`lettuce wrap` would match `get me a lettuce wrap ASAP` as
+`[0 0 0 1 1 0]`. 
+
+These regexes are processed identically to the regular
+regex patterns directly specified in the training data and can be used
+either with the [RegexFeaturizer](../nlu/components#regexfeaturizer)
+or with the [RegexEntityExtractor](../nlu/components#regexentityextractor).
+
+**note**
+
+If you are using lookup tables in combination with the RegexFeaturizer, there must be a few examples of matches  
+in your training data. Otherwise the model will not learn to use the
+lookup table match features.
+
+**warning**
+
+You have to be careful when you add data to the lookup table.  
+For example, if there are false positives or other noise in the table,
+this can hurt performance. So make sure your lookup tables contain
+clean data.
+
+## Conversation Training Data
+
+Stories and rules are both representations of conversations between a user and a conversational
+assistant. They are used to train the dialogue management
+model. **Stories** are used to train a machine learning model to identify patterns
+in conversations and generalize to unseen conversation paths.
+**[Rules](../core/rules)** describe parts of conversations that should always
+follow the same path and are used to train the
+[RulePolicy](../core/policies#rule-policy). 
+
+
+### Stories
+
+Stories are composed of:
+  
+  - name: The name is arbitrary and not used in training; 
+    you can use it as a human-readable reference for the story.
+  - metadata: arbitrary and optional, not used in training, 
+    you can use it to store relevant information about the story
+    like e.g. the author
+  - a list of ``steps``: The user messages and actions that make up the story
```

Comment:
name: here is confusing because it makes me think that is the key. maybe we do it by keys? 

```
Stories include the following keys: 

  - `story` (required): The story's name. The name is not used in training; 
     use it as a human-readable reference for the story.
  - `metadata` (optional): Extra information about the story. Not used in training; 
     you can use it to store relevant information about the story, e.g. the author.
  - a list of `steps` (required): The user messages and actions that make up the story.

Reply:
oh wow did not even see that I misnamed the name. Thanks